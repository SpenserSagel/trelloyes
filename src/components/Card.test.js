import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card.js';

describe('cardTest',() => {
    it('rendersWithoutCrash', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card/>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('rendersAsExpected', () => {
        const tree = renderer
            .create(<Card title="testTitle" content="testContent"/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});