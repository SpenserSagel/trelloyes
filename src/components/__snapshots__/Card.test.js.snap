// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cardTest rendersAsExpected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    testTitle
  </h3>
  <p>
    testContent
  </p>
</div>
`;
